strings/languages/ru/strings.json

```diff
@@ -146,10 +146,10 @@
     "calendar": {
       "sameDay": "[Сегодня]",
       "nextDay": "[Завтра]",
-      "nextWeek": "дддд",
+      "nextWeek": "dddd",
```

Comment:
Since these should not be translated they should be removed from the `strings/languages/en/strings.json` file.

Reply:
I suppose to delete `sameElse` as well ?